pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -186,71 +185,74 @@ def images() -> DownloadableImageCollection:
     return DownloadableImageCollection({**content_images, **style_images})
 
 
+class SkipSmallIterableDataset(IterableDataset):
```

Comment:
After some more thought, lets just call this `Dataset` here

```suggestion
class Dataset(IterableDataset):
```

`SkipSmallIterableDataset` is only partially descriptive, since we also cycle through the data up to a given number of samples.